charts/vault-cred/templates/deployment.yaml

```diff
@@ -42,6 +43,12 @@ spec:
               value: "{{ .Values.env.logLevel }}"
             - name: VAULT_ADDR
               value: "{{ .Values.vault.vaultAddress }}"
+            - name: VAULT_ADDR2
+              value: "{{ .Values.vault.vaultAddress2 }}"
+            - name: VAULT_ADDR3
+              value: "{{ .Values.vault.vaultAddress3 }}"
```

Comment:
keep VAULT_CLUSTER_ADDR, add the vault service address
VAULT_SERVER_ADDRESSES - add all 1,2,3 valut addresses.

use first address like it is used currently with VAULT_ADDR